<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateChecklistPage</class>
 <widget class="QWidget" name="CreateChecklistPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#backButton {
	background: transparent;
	border: none;
	font-family: Open Sans;
	font-size: 24px;
	line-height: 34px;
	font-weight: 600;
	letter spacing: 0%
}

QLabel {
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 600;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
}

QLineEdit {
	color: #1A1A1A;
	background: white;
	font-family: Open Sans;
	font-weight: 400;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1px solid #CCCCCC;
	padding-left: 12px;
	padding-right: 12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Create Checklist</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/Resources.qrc">
         <normaloff>:/images/nav_back.svg</normaloff>:/images/nav_back.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: white;
	border: 2px solid #1A1A1A;
	border-radius: 8px;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	color: #1A1A1A;
}

QPushButton:hover {
    background-color: #D6ECFF;
}

QPushButton:pressed {
    background-color: #B7DCFF;
    padding-top: 2px;
	padding-left: 2px;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: #FDB515;
	border: none;
	border-radius: 8px;
	font-family: Open-Sans;
	font-weight: 500;
	font-size: 18px;
	color: white;
}

QPushButton:hover {
    background-color: #E6A512;
}

QPushButton:pressed {
    background-color: #CC8F0F;
    padding-top: 2px;
	padding-left: 2px;
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="container">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>815</width>
        <height>365</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="containerLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="TitleContainer" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="titleEdit">
            <property name="minimumSize">
             <size>
              <width>522</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>522</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="QuestionContainer" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="questionsLabel">
            <property name="text">
             <string>Questions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="questionItems" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addQuestionButton">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>183</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background: white;
	border: 2px dashed black;
	border-radius: 8px;
	font-family: Open-Sans;
	font-weight: 500;
	font-size: 18px;
	color: black;
}

QPushButton:hover {
    background-color: #D6ECFF;
}

QPushButton:pressed {
    background-color: #B7DCFF;
    padding-top: 2px;
	padding-left: 2px;
}</string>
         </property>
         <property name="text">
          <string>Add Question</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
